sources/editor/Stride.Core.Assets.Editor/Services/VisualStudioService.cs

```diff
@@ -94,6 +98,14 @@ public static async Task<Process> StartVisualStudio(SessionViewModel session, ID
             }
         }
 
+        private static EditorInfo ToEditorInfo(IDEInfo ideInfo)
```

Comment:
This is a really simple method and only has a single callsite, best to move its logic to the callsite instead, no need to fragment logic unnecessarily